composer.json

```diff
@@ -37,12 +38,8 @@
         "twig/twig": "^1.34 || ^2.0"
     },
     "require-dev": {
-        "sonata-project/doctrine-orm-admin-bundle": "^3.0",
```

Comment:
shouldn't this be moved to require?

Reply:
It is not needed for this project. There is no class usage for anything